From 2b5e4a0fe2593ba213355fa90503cee16e9b0b7a Mon Sep 17 00:00:00 2001
From: Nicolas Schichan <nschichan@freebox.fr>
Date: Wed, 12 Jul 2017 15:40:25 +0200
Subject: [PATCH 1089/1345] net: mvpp2x: fix misidented return statement in
 mv_pp2x_bm_pools_init().

commit  3e620dc05d461e50339af087caf292882b6c81cb from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Fix a warning reported by recent versions of GCC used by FREEBOX.

Change-Id: Ic1a0d1ba463190ca4502a782d5f674d4efec5f13
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/41677
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
index f28a91a..f35c723 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
@@ -536,7 +536,7 @@ static int mv_pp2x_bm_pools_init(struct platform_device *pdev,
 	dev_err(&pdev->dev, "failed to create BM pool %d, size %d\n", i, size);
 	for (i = i - 1; i >= 0; i--)
 		mv_pp2x_bm_pool_destroy(&pdev->dev, priv, &priv->bm_pools[i]);
-		return err;
+	return err;
 }
 
 static int mv_pp2x_bm_init(struct platform_device *pdev, struct mv_pp2x *priv)
-- 
1.7.9.5

